docs/en/guides/pipeline-integrations/administration/config-staging-and-rollout.md

```diff
@@ -6,3 +6,489 @@ lang: en
+
+Each toolkit project contains an independent configuration which holds all the settings for that project. If you are managing a lot of projects, this means that you need to manage a lot of configurations. For this reason, Toolkit contains a number of different tools and ways to handle configurations, both for individual projects and for the studio life cycle and configuration evolution. This section outlines some of those best practices and shows practical steps how to set up your config.
+
+Depending on the needs of your studio, different levels of complexity may be relevant. Toolkit offers three different approaches and we'll explain each one of them in detail:
+
+-   The most straight forward approach is to basically copy the config from the previous project when you set up a new project. This is good if you are a small studio and don't have a massive amount of projects.
+-   If you have a higher project turnover and if you run more than one project in parallel, the next level of integration that we recommend involves  `git`  version control. Toolkit has got native support for git and once you are up and running with a git based workflow you have a single configuration for your studio and are tracking all the changes you are making to that configuration over time. Each project can safely pull in configuration changes as and when they need to.
+-   If you are running a large scale facility, it may be worth considering a setup where a single configuration is directly connected to all the different currently running projects in the studio. A single change to this configuration will have an immediate effect on all the projects.
+
+In the following sections we'll describe the different approaches in detail.
+
+## Inheriting the config from your previous project
+
+This is the simplest approach to configuration management. For the very first time you set up a project with Toolkit, you download our default configuration. You then set up that first project to match your pipeline settings, file system locations etc. When the project is running, you run commands such as  `tank updates`  to update the app versions you are running.
+
+When your second project comes around, you don't want to start with the default config again - instead, during the project setup phase, pick the config from your first project as the starter for your new project. This means that all the changes, tweaks and improvements you did for the first project will be taken advantage of in the second project as well. For the third project, use the second project's config etc.
+
+![](images/config-staging-and-rollout/copy_config.png)
+
+This is a very simple way to gradually evolve the configuration over time. Changes and improvements will flow from project to project in an ad hoc fashion. The first time you run the  `setup_project`  command, just hit enter when the setup process prompts for the configuration to use. This will download and install the default configuration.  
```

Comment:
```suggestion
This is a very simple way to gradually evolve the configuration over time. Changes and improvements will flow from project to project in an ad hoc fashion. The first time you run the `setup_project` command, just hit enter when the setup process prompts for the configuration to use. This will download and install the default configuration.  
```